### PR TITLE
Fix minikube image 'project:hello-minikube-zero-install' has been suspended.

### PR DESCRIPTION
**Pull request by lou-lan**
See #20530, the description for pull image 'project:hello-minikube-zero-install' seems wrong, and this PR is going to fix this.

### CODE REVIEW

**Comment by Swoorup** [comment]
Can confirm this new url fixes the issue.

File: content/de/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Deployments sind die empfohlene Methode zum Verwalten der Erstellung und Skalier
 Der Pod führt einen Container basierend auf dem bereitgestellten Docker-Image aus.
 
     ```shell
-    kubectl create deployment hello-node --image=gcr.io/hello-minikube-zero-install/hello-node
+    kubectl create deployment hello-node --image=k8s.gcr.io/echoserver:1.10
```
Comment (bradtopol) [inline content/de/docs/tutorials/hello-minikube.md:78]: Can you make this echoserver:1.4 so it matches what is on https://minikube.sigs.k8s.io/docs/start/

**Comment by lou-lan** [comment]
> Lou-lan thank you for your contribution. Can you make this echoserver:1.4 so it matches what is on https://minikube.sigs.k8s.io/docs/start/
> 
> Also you need break this up into multiple PRs where you have one per language. In general it causes problems later if we have a single PR crossing over multiple languages.

Thanks for the suggestion, I will modify it and decompose it into multiple PRs.

**Comment by tengqm** [comment]
@lou-lan are you still working on this?

**Pull request closed without merging**